modules/ROOT/pages/dataweave-formats.adoc

```diff
@@ -537,6 +537,12 @@ The DataWeave (weave) format is the canonical format for all transformations.
 This format can help you understand how input data is interpreted before it is
 transformed to a new format.
 
+[WARNING]
+====
+This format is intended only as a help for debugging the results of DataWeave transformations.
```

Comment:
This format helps debug the results of DataWeave transformations.

Reply:
I like my version because it is more assertive of the intended usage.